metadata/metadata.go

```diff
@@ -1572,6 +1832,20 @@ func (serv *MetadataServer) genericCreate(ctx context.Context, res Resource, ini
 	return &pb.Empty{}, nil
 }
 
+func isEquivalent(this Resource, other Resource) bool {
```

Comment:
Is the idea that we will expand it to other Resources too?

Reply:
Eventually we could do it; I think it will really make sense for providers

Reply:
But right now the equivalnce check if for finding equivalent variants